src/main/java/chess/controller/LobbyController.java

```diff
@@ -0,0 +1,73 @@
+package chess.controller;
+
+import chess.domain.game.ChessGame;
+import chess.domain.piece.Color;
+import chess.dto.*;
+import chess.service.LobbyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@Controller
+public class LobbyController {
+    private final LobbyService lobbyService;
+
+    @Autowired
+    public LobbyController(LobbyService lobbyService) {
+        this.lobbyService = lobbyService;
+    }
+
+    @GetMapping("/")
+    public String lobby() {
+        return "lobby";
+    }
+
+    @PostMapping("/lobby/new")
+    @ResponseBody
+    public ResponseEntity newGame(@RequestBody @Valid TitleDTO titleDTO, BindingResult error) {
+        if (error.hasErrors()) {
+            return ResponseEntity.badRequest().body(error.getAllErrors().get(0).getDefaultMessage());
+        }
+        return ResponseEntity.ok(new RoomIdDTO(lobbyService.newGame(titleDTO.getTitle())));
+    }
+
+    @PostMapping("/findRoomId")
+    @ResponseBody
+    public ResponseEntity findRoomId(@RequestBody @Valid TitleDTO titleDTO, BindingResult error) {
+        String title = titleDTO.getTitle().replaceAll("%20", " ");
+        if (error.hasErrors()) {
+            return ResponseEntity.badRequest().body(error.getAllErrors().get(0).getDefaultMessage());
+        }
+        return ResponseEntity.ok(new RoomIdDTO(lobbyService.findRoomId(title).orElseThrow(() -> new EmptyResultDataAccessException(0))));
+    }
+
+    @PostMapping("/isDuplicate")
+    @ResponseBody
+    public ResponseEntity isDuplicate(@RequestBody TitleDTO titleDTO) {
```

Comment:
전 명시적으로 리턴타입을 정해주는 편이에요 :) 
```suggestion
    public ResponseEntity<DuplicateDTO> isDuplicate(@RequestBody TitleDTO titleDTO) {
```